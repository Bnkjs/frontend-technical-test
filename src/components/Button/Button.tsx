import React, {FC} from "react"
import style from '../../styles/Home.module.scss'

export const Button = (props) => {
  return(
    <button disabled={props.isDisabled} ref={props.btnRef} onClick={props.onClick} className={style.btnSend}>Envoyer</button>
  )
}