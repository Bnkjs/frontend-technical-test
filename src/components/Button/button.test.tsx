import renderer from 'react-test-renderer';
import { Button } from "./Button"

describe('should render correctly Button', () =>{
 const component = renderer.create(<Button/>)
 const tree = component.toJSON()
 expect(tree).toMatchSnapshot()
})
