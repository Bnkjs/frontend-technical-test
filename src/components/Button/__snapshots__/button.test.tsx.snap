// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<button
  className="btnSend"
>
  Envoyer
</button>
`;
